### ISSUE TITLE
Example using the 2.0.0 version

### ISSUE THREAD

**Issue by claudiuconstantin** [sureshchahal/angular2-adal#67]
Can you provide an example using the new angular 4 version of the package? Seems that some breaking changes were introduced lately

**Comment by sureshchahal** [comment]
It is same but used angular 4

**Comment by claudiuconstantin** [comment]
Well, I use `ng2-adal: ^2.0.0`, importing `import { AdalService } from "ng2-adal/core"` and I'm getting this:

`Module not found: Error: Can't resolve 'ng2-adal/core'`

**Comment by rjwijnen** [comment]
Use ng2-adal/dist/core instead

**Comment by claudiuconstantin** [comment]
Ok, that worked. This should be specified in the package description, a lot of people will have this issue after upgrade.
You can close this one.

**Comment by skorunka** [comment]
I had the same problem as @claudiuconstantin . I appreciate your hard work guys, but please, when you release a new version, with a breaking changes, please make sure you outline the breaking changes somewhere. Thank you.

**Comment by FabianGosebrink** [comment]
Had the same problems. Missed the changelog and the release tag here in github. Nice work but remember that you have to reflect the changes for you users. Thanks :) Keep it up.

**Comment by jderus** [comment]
Had the same problem.
Would be nice to see the example updated, and have it also include proper usage of AuthHttp.

**Comment by hitmanpc** [comment]
Yeah it took me 2 days to find this ... thanks for updating this issue.

**Comment by claudiuconstantin** [comment]
I created a [pull request](https://github.com/sureshchahal/angular2-adal/pull/69) with the improved Readme file, check it out

**Comment by ranveeraggarwal** [comment]
I have updated the [example](https://github.com/ranveeraggarwal/angular-adal-quickstart) to use the latest version of this package and Angular 4.
